src/main/resources/io/viash/runners/nextflow/arguments/_processOutputValues.nf

```diff
@@ -1,12 +1,5 @@
-Map _processOutputValues(Map outputs, Map config, String id, String key) {
+Map _checkValidOutputArgument(Map outputs, Map config, String id, String key) {
   if (!workflow.stubRun) {
-    config.allArguments.each { arg ->
-      if (arg.direction == "output" && arg.required) {
-        assert outputs.containsKey(arg.plainName) && outputs.get(arg.plainName) != null : 
-          "Error in module '${key}' id '${id}': required output argument '${arg.plainName}' is missing"
-      }
-    }
-
```

Comment:
Is this check still happening somewhere?

Reply:
Good catch! Added it back with e46ed7ffa5e49bb8bec76b6b8b04d6f07190d20c